components/self-service/new-content-form.tsx

```diff
@@ -13,6 +19,63 @@ import HelpIcon from '@mui/icons-material/Help';
 import frontendConfig from "../../frontend.config.json"
 import { useState } from 'react';
 
+
+const applicationCategories = [
+  'API Management',
+  'Accounting',
+  'Application Delivery',
+  'Application Server',
+  'Automation',
+  'Backup & Recovery',
+  'Business Intelligence',
+  'Business Process Management',
+  'Capacity Management',
+  'Cloud Management',
+  'Collaboration/Groupware/Messaging',
+  'Configuration Management',
+  'Container Platform / Management',
+  'Content Management/Authoring',
+  'Customer Relationship Management',
+  'Data Store',
+  'Database & Data  Management',
+  'Developer Tools',
+  'Enterprise Resource Planning',
+  'Identity Management',
+  'Integration',
+  'Logging & Metrics',
+  'Management',
+  'Messaging',
+  'Middleware',
+  'Migration',
+  'Mobile Application Development Platform (MADP)',
+  'Monitoring',
+  'Network Management',
+  'Networking',
+  'Operating System',
+  'Other',
+  'Performance Management',
+  'Policy Enforcement',
+  'Programming Languages & Runtimes',
+  'Scheduling',
+  'Search',
+  'Security',
+  'Storage',
+  'Virtualization Platform',
+  'Web Services',
+];
+
+const usageTypeHelperText = [
+  ["Automation execution environment", "Automation execution environments are self-contained images that are used to execute Ansible automation with the Ansible Automation Platform, containing a version of the ansible-core runtime, and a specific set of Ansible content."],
```

Comment:
Do you know if this list has an order? usually, they will choose one of `Component image`, `Operator image`, `Standalone image` and `Operator bundle`, we better make them at the top of drop-down list so no need to scroll down to the bottom

Reply:
Yep, it can be reordered, however, here it is not used in a drop-down, but as radio buttons instead, which are all directly visible to the user right away and they need to scroll down anyway to fill in the rest of the form, which is why I put them there in the original order.
Should I reorder the radio buttons or remake it into a drop-down with the new ordering?

Reply:
Don't think we even support the other types, besides the ones that Ximin listed. So maybe we can list the ones that we support for now

Reply:
I see, I have removed all the types except for `Component image`, `Operator image`, `Standalone image` and `Operator bundle`